src/Database/DBA.php

```diff
@@ -524,20 +521,20 @@ public static function count($table, array $condition = [], array $params = []):
 	 * - [table1, table2, ...]
 	 * - [schema1 => table1, schema2 => table2, table3, ...]
 	 *
-	 * @param string|array $tables
+	 * @param array $tables Table names
 	 * @return string
 	 */
-	public static function buildTableString($tables): string
+	public static function buildTableString(array $tables): string
 	{
-		if (is_string($tables)) {
-			$tables = [$tables];
-		}
-
 		$quotedTables = [];
 
 		foreach ($tables as $schema => $table) {
 			if (is_numeric($schema)) {
-				$quotedTables[] = self::quoteIdentifier($table);
+				$str = '';
```

Comment:
Well, some quick'n'dirty code here. Normally only 1 or 2 elements should be expected here.